feature_engine/mathematical_transformers.py

```diff
@@ -0,0 +1,86 @@
+from feature_engine.base_transformers import BaseNumericalTransformer
+
+
+class AdditionTransformer(BaseNumericalTransformer):
+    """
+    The AdditionTransfomer() applies basic mathematical operations using aggregation of features.
+
+    Parameters
+    ----------
+
+    variables: list, default=None
+        The list of numerical variables to be transformed. If None, the transformer
+        will find and select all numerical variables.
+
+    math_operations: list, default=None
+        The list of basic math operations to be used in transformation.
+
+        Each operation should be a string and must be one of elements
+        from list: ['sum', 'prod', 'mean', 'std', 'max', 'min']
+
+        Each operation will result in operation column in result dataset.
+
+        If None, the transformer will calculate each of permitted
+        operations: ['sum', 'prod', 'mean', 'std', 'max', 'min']
+    """
+
+    def __init__(self, variables=None, math_operations=None):
+        self.variables = variables
+        self.math_operations = math_operations
+        self.math_operations_permitted = ['sum', 'prod', 'mean', 'std', 'max', 'min']
+
+        if self.math_operations is None:
+            self.operations_ = self.math_operations_permitted
```

Comment:
we can't include attributes with trailing score outside the fit() method, because the is_fitted() method from scikit-learn, which we use in the transform method  checks exactly for trailing underscores.

Reply:
right, I'll change this.